src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesignerOptionService.cs

```diff
@@ -154,6 +154,7 @@ internal DesignerOptionCollection(DesignerOptionService service, DesignerOptionC
 
                 if (Parent != null)
                 {
+                    parent._properties = null;
```

Comment:
Do we have a test that ensures this will continue caching the children afterwards? It should, but just want to make sure we don't regress any behavior.